chat-commands.js

```diff
@@ -710,7 +710,7 @@ const commands = {
 	},
 	makegroupchathelp: [
 		`/makegroupchat [roomname] - Creates an invite-only group chat named [roomname]. Requires global voice or roomdriver+ in a public room to make a groupchat.`,
-		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a public room you have staff in.`,
+		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a room you have staff in.`,
```

Comment:
This is wrong, only trusted users can use it. I personally think "public" was fine. There's no real need to advertise that trusted users can make subroom groupchats of private rooms they staff.

Reply:
but saying that it "can only be used in a public room..." is directly incorrect

Reply:
Yeah, but it's at least a form of incorrect I don't mind.